ClientKit/Display.cs

```diff
@@ -0,0 +1,350 @@
+
+        /// <summary>
+        /// Height of viewport in pixels.
+        /// </summary>
+        public ViewportDimension Height { get; set; }
+
+        public override string ToString()
+        {
+            return String.Format("(left: {0}, bottom: {1}, width: {2}, height: {3})",
+                Left, Bottom, Width, Height);
+        }
+    }
+
+    /// <summary>
+    /// OSVR display configuration object. Allows you to query viewer,eye,surface matrices.
+    /// </summary>
+    public class DisplayConfig : IDisposable
+    {
+        private SafeDisplayConfigHandle mHandle;
+        internal DisplayConfig(SafeDisplayConfigHandle handle)
+        {
+            mHandle = handle;
+        }
+
+        private void CheckSuccess(Byte returnCode, string message)
+        {
+            if(returnCode != OSVR.ClientKit.ClientContext.OSVR_RETURN_SUCCESS)
+            {
+                throw new InvalidOperationException(message);
+            }
+        }
+
+        /// <summary>
+        /// A display config can have one (or theoretically more) viewers
+        /// </summary>
+        public ViewerCount GetNumViewers()
+        {
+            ViewerCount ret;
+            CheckSuccess(
+                DisplayConfigNative.osvrClientGetNumViewers(mHandle, out ret),
+                "[OSVR] DisplayConfig.GetNumViewers(): native osvrClientGetNumViewers call failed.");
+            return ret;
+        }
+
+        /// <summary>
+        /// Get the center of projection/"eye point" for a viewer in a display config.
+        /// Note that there may not necessarily be any surfaces rendered from this pose
+        /// (it's the unused "center" eye in a stereo configuration) so only use this if
+        /// it makes integration into your engine or existing applications (not
+        /// originally designed for stereo) easier.
+        /// </summary>
+        /// <param name="viewer">Viewer ID</param>
+        public Pose3 GetViewerPose(ViewerCount viewer)
+        {
+            Pose3 ret;
+            CheckSuccess(
+                DisplayConfigNative.osvrClientGetViewerPose(mHandle, viewer, out ret),
+                "[OSVR] DisplayConfig.GetViewerPose(): native osvrClientGetViewerPose call failed.");
+
+            return ret;
+        }
+
+        /// <summary>
+        /// Each viewer in a display config can have one or more "eyes" which
+        /// have a substantially similar pose.
+        /// </summary>
+        /// <param name="viewer">Viewer ID</param>
+        public EyeCount GetNumEyesForViewer(ViewerCount viewer)
+        {
+            EyeCount ret;
+            CheckSuccess(
+                DisplayConfigNative.osvrClientGetNumEyesForViewer(mHandle, viewer, out ret),
+                "[OSVR] DisplayConfig.GetNumEyesForViewer(): native osvrClientGetNumEyesForViewer call failed.");
+            return ret;
+        }
+
+        /// <summary>
+        /// Get the center of projection/"eye point" for a viewer in a display
```

Comment:
this whole comment - it's the same as the previous one, rather than being the one of the ViewerEye